packages/webex/package.json

```diff
@@ -14,6 +14,11 @@
     "Taymoor Khan <taykhan@cisco.com>"
   ],
   "main": "dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./meetings": "./dist/meetings.js",
+    "./package": "./package.json"
```

Comment:
It's imported in the tests of webex package -> https://github.com/mkesavan13/webex-js-sdk/blob/bd9ec6ec762e457b6ef6b97e8932f659bb1579e6/packages/webex/test/unit/spec/webex.js#L7

Since we specifically mention exports now, we need to mention this as well.

Reply:
@mkesavan13  lets create a ticket to fix this , is there a way to remove the self import, other packages do the same but there was no need to add the exports for those